v1.0.0/bash/rollback.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Pause all pipelines in the pipedream
+gocd-pause-and-cancel-pipelines \
+  --pause-message="This pipeline is being rolled back, please check with team before un-pausing." \
+  <PIPELINE_FLAGS>
+
+# Get sha from the given pipeline run to deploy to all pipedream pipelines.
+sha=$(gocd-sha-for-pipeline --material-name="${ROLLBACK_MATERIAL_NAME}")
+
+gocd-emergency-deploy \
+  --commit-sha="${sha}" \
+  --deploy-stage="${ROLLBACK_STAGE}" \
+  --pause-message="This pipeline was rolled back, please check with team before un-pausing." \
+  <PIPELINE_FLAGS>
```

Comment:
another option, would be to put the arguments in an environment variable named "ARGS", and add a snippet like

```sh
#!/bin/bash
if [[ "${ARGS:-}" ]]; then
  # shellcheck disable=SC2086
  set -- $ARGS   # note: no quoting, for word expansion
fi

...
"$@"
```

Reply:
Love this approach - testings locally running the script with set -x and output looks correct.